build.gradle

```diff
@@ -374,4 +376,143 @@ buildRpm {
 buildDeb {
     arch = 'all'
     archiveName "${packageName}-${version}.deb"
+}
+
+String bwcVersion = "1.13.1.0";
+String baseName = "securityBwcCluster"
+String bwcFilePath = "src/test/resources/bwc"
+
+2.times {i ->
```

Comment:
can you add a comment here please explaining why we need two clusters here (just like you did below)